README.md

```diff
@@ -84,3 +84,17 @@ yarn dev:burnr
 
 
 ## [Deployments and releases](./DEPLOY-RELEASE.md)
+
+
+## Substrate Connect Extension
+
+A Browser extension that keeps the latest state of well known substrate-based chains' specs and bootnodes (Polkadot, Kusama, Rococo, Westend) synced across tabs - using Substrate Connect and Smoldot light client; 
```

Comment:
> A Browser extension that keeps the latest state of well known substrate-based chains' specs and bootnodes (Polkadot, Kusama, Rococo, Westend) synced across tabs

I don't think that this is true anymore,. Unfortunately, the extension no-longer keeps things in sync across tabs. The extension does let the user know how the light-client is being used across the different tabs by displaying open connections and the tabs that are using the light-client, etc. However, it's not accurate to say that the extension keeps the latest state of well known substrate-based chains synced across tabs. That used to be the case, but unfortunately it no longer is.